tests/foreman/cli/test_organization.py

```diff
@@ -115,142 +113,62 @@ def test_verify_bugzilla_1078866(self):
-        """Create organization with valid name and description only
-
-        :id: b28c95ba-918e-47fe-8681-61e05b8fe2ea
-
-        :expectedresults: organization is created
-
-        :CaseImportance: Critical
-        """
-        for name, desc in zip(valid_org_names_list(), valid_data_list()):
-            with self.subTest(name + desc):
-                org = make_org({
-                    'description': desc,
-                    'name': name,
-                })
-                self.assertEqual(org['name'], name)
-                self.assertEqual(org['description'], desc)
-
-    @tier1
-    @upgrade
-    def test_positive_create_with_name_label_description(self):
+    def test_positive_CRD(self):
         """Create organization with valid name, label and description
 
-        :id: 9a1f70f6-fb5f-4b23-9f7e-b0973fbbba30
+        :id: 35840da7-668e-4f78-990a-738aa688d586
 
-        :expectedresults: organization is created
+        :expectedresults: organization is created with attributes
 
         :CaseImportance: Critical
-        """
-        for description in valid_data_list():
-            with self.subTest(description):
-                label = gen_string('alpha')
-                name = gen_string('alpha')
-                org = make_org({
-                    'description': description,
-                    'label': label,
-                    'name': name,
-                })
-                self.assertEqual(org['description'], description)
-                self.assertEqual(org['label'], label)
-                self.assertEqual(org['name'], name)
-
-    @tier1
-    def test_positive_list(self):
-        """Check if Org can be listed
 
-        :id: bdd26bb3-e3d2-4a5c-8be7-fb12c1114ccc
-
-        :expectedresults: Org is listed
-
-        :CaseImportance: Critical
+        create read
         """
-        org = make_org()
-        result_list = Org.list({'search': 'name=%s' % org['name']})
-        self.assertTrue(len(result_list) > 0)
-        self.assertEqual(result_list[0]['name'], org['name'])
-
-    @tier1
-    def test_positive_search_scoped(self):
-        """Check if scoped search work properly for organization entity
-
-        :id: d66ebe1d-aba1-4042-87e4-7de8ea0f8fc8
-
-        :customerscenario: true
-
-        :expectedresults: Necessary organization is listed
+        # Create
+        name = valid_org_names_list()[0]
+        label = valid_labels_list()[0]
+        desc = valid_data_list()[0]
+        org = make_org({
+            'name': name,
+            'label': label,
+            'description': desc,
+            })
+        self.assertEqual(org['name'], name)
+        self.assertNotEqual(org['name'], org['label'])
+        self.assertEqual(org['label'], label)
+        self.assertEqual(org['description'], desc)
 
-        :BZ: 1259374
+        # List
+        result = Org.list({'search': 'name=%s' % org['name']})
+        self.assertTrue(len(result) > 0)
```

Comment:
will below code add more value, what you think ?
 
` self.assertTrue(len(result) == 2)` assuming default org will exist.